sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -1265,3 +1501,14 @@ def display_data(self):
                                    tableSpec)
       res['table'] = DisplayDataItem(tableSpec, label='Table')
     return res
+
+
+@experimental()
+class ReadFromBigQuery(PTransform):
+  def __init__(self, *args, **kwargs):
```

Comment:
Since `ReadFromBigQuery` is the user-facing transform, it should have all the Pydoc.
That being said, I'll defer @chamikaramj whether we want to expose this already.

Reply:
+1